packages/wonder-blocks-core/util/types.ts

```diff
@@ -87,26 +89,26 @@ type roles =
     | "window";
 
 type IdRef = string;
-type IdRefList = IdRef | Array<IdRef>;
+type IdRefList = IdRef; //  | Array<IdRef>;
 
 // Source: https://www.w3.org/WAI/PF/aria-1.1/states_and_properties
 // TODO(kevinb): convert to disjoint union of exact object types keyed on role
 // eslint-disable-next-line flowtype/require-exact-type
-export type AriaProps = {|
+export type AriaProps = {
     role?: roles,
     "aria-activedescendant"?: IdRef,
     "aria-atomic"?: "false" | "true",
     "aria-autocomplete"?: "both" | "inline" | "list" | "none",
     "aria-busy"?: "true" | "false",
-    "aria-checked"?: "false" | "mixed" | "true" | "undefined",
+    "aria-checked"?: "false" | "mixed" | "true" | boolean,
     "aria-controls"?: IdRefList,
     "aria-describedat"?: string, // URI
     "aria-describedby"?: IdRefList,
     "aria-disabled"?: "false" | "true",
     "aria-dropeffect"?: "copy" | "execute" | "link" | "move" | "none" | "popup",
-    "aria-expanded"?: "false" | "true" | "undefined",
+    "aria-expanded"?: "false" | "true" | boolean,
```

Comment:
These changes were to bring these type definitions into line with TypeScript's aria type definitions.  I wonder if there's a way we just reuse those type definitions instead of creating our own.  🤔